server/@types/bapv.d.ts

```diff
@@ -134,11 +135,7 @@ export type VisitSessionData = {
   }
   visit?: VisitSlot
   visitors?: VisitorListItem[]
-  additionalSupport?: {
-    required: boolean
-    keys?: string[]
-    other?: string
-  }
+  visitorSupport?: VisitorSupport[]
```

Comment:
Changed the format of the data we store in the session to match what gets saved to the visit scheduler to avoid converting back/forth. So, using array of `VisitorSupport` items and just checking length rather than separate `required` value.